client/src/modules/list/components/Items/InputBar/InputBar.jsx

```diff
@@ -143,8 +146,6 @@ class InputBar extends Component {
     if (isInputEmpty) {
       this.hideForm();
     }
```

Comment:
when input loses focus listener is still invoking, I think that on blur we should always remove the listener, not only when the input isn't empty.